sketches/node-navigator.js

```diff
@@ -27,11 +27,18 @@ class Hashtag {
 	setDragged(b) { this.isDragged = b; }
 
 	display() {
-		this.p5.fill(211, 103, 60);
-		this.p5.noStroke();
-		this.p5.textAlign(this.p5.CENTER, this.p5.CENTER);
-		this.p5.textFont(font, this.textSize);
-		this.p5.text(this.name, this.position.x, this.position.y, hash_radius * 2, hash_radius * 2);
+		if (this.p5.mouseX >= this.position.x - 50 &&
+			this.p5.mouseX <= this.position.x + 50 &&
+			this.p5.mouseY >= this.position.y - (this.textSize * 2) &&
+			this.p5.mouseY <= this.position.y) {
```

Comment:
can u turn this into a hover function?

if(isHovering(...)) {
}

Reply:
Done!